### PR TITLE
PR to upgrade `react-native-webview` library to `13.6.3`

### PR DESCRIPTION
**Pull request by siddarthkay**
### Summary

We need to upgrade `react-native-webview` as a pre-requisite for upgrading `react-native` to `0.73.x`
else we face this issue while building iOS : https://github.com/react-native-webview/react-native-webview/issues/3233

### Testing notes
 `react-native-webview` is currently used in legacy code and primarily for our browser implementation.

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
- browser

##### Functional
- dapps / app browsing

status: ready

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]

**Review by OmarBasem — APPROVED** [review]

**Comment by siddarthkay** [comment]
Hi @status-im/mobile-qa : This PR upgrades webview library used by browser and the usage is in legacy codebase, so I don't think this needs testing. 
I've run E2E once more just in case, will merge after.

**Pull request merged**
